scala/event-driven/events-to-snapshot/docker-compose.yml

```diff
@@ -0,0 +1,131 @@
+version: "2"
+services:
+  tools:
+    image: gitlab.pipefy.net:8443/pipefy/docker-images/kafka-cp:latest
+    entrypoint: /bin/bash -c "tail -f /dev/null"
+    networks:
+      - pipes-stream
+
+  app:
+    image: mozilla/sbt
+    environment:
+      - JAVA_OPTIONS=-Xmx300m
+    command: /bin/sh -c "tail -f /dev/null"
+    volumes:
+      - .:/app
+    networks:
+      - pipes-stream
+    depends_on:
+      - zookeeper
+      - broker
+      - kafka-schema-registry
+
+  test:
+    image: mozilla/sbt
+    environment:
+      - JAVA_OPTIONS=-Xmx300m
+      - PFY_STREAMS_PIPE_ES_APP_ENV=test
+    command: /bin/sh -c "tail -f /dev/null"
+    volumes:
+      - .:/app
+
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.3.0
+    networks:
+      - pipes-stream
+    hostname: zookeeper
+    ports:
+      - "2181:2181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-enterprise-kafka:5.2.1
+    networks:
+      - pipes-stream
+    hostname: broker
+    depends_on:
+      - zookeeper
+    ports:
+      - "29092:29092"
+      - "9092:9092"
+    environment:
+      KAFKA_BROKER_ID: 1
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_ADVERTISED_HOST_NAME: "broker"
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://localhost:9092
+      KAFKA_METRIC_REPORTERS: io.confluent.metrics.reporter.ConfluentMetricsReporter
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: broker:29092
+      CONFLUENT_METRICS_REPORTER_ZOOKEEPER_CONNECT: zookeeper:2181
+      CONFLUENT_METRICS_REPORTER_TOPIC_REPLICAS: 1
+      CONFLUENT_METRICS_ENABLE: "true"
+      CONFLUENT_SUPPORT_CUSTOMER_ID: "anonymous"
+
+  kafka-schema-registry:
+    image: confluentinc/cp-schema-registry:5.3.0
+    networks:
+      - pipes-stream
+    hostname: kafka-schema-registry
+    ports:
+      - "8081:8081"
+    environment:
+      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: PLAINTEXT://broker:29092
+      SCHEMA_REGISTRY_HOST_NAME: kafka-schema-registry
+      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:8081
+    depends_on:
+      - zookeeper
+      - broker
+  
+  # kafka-schema-registry-ui:
```

Comment:
Those comments are important? Can we remove it? or it is some sort of example? If that's the case, what do you think about moving the example to some `.md` file and explain it \o/